sandbox/modules/identity/src/main/java/org/opensearch/identity/IdentityPlugin.java

```diff
@@ -92,6 +105,37 @@ private static boolean isEnabled(final Settings settings) {
         return settings.getAsBoolean(ConfigConstants.IDENTITY_ENABLED, false);
     }
 
+    @Override
+    public List<RestHandler> getRestHandlers(
+        Settings settings,
+        RestController restController,
+        ClusterSettings clusterSettings,
+        IndexScopedSettings indexScopedSettings,
+        SettingsFilter settingsFilter,
+        IndexNameExpressionResolver indexNameExpressionResolver,
+        Supplier<DiscoveryNodes> nodesInCluster
+    ) {
+        final List<RestHandler> handlers = new ArrayList<>(1);
+        handlers.add(new RestCreateUserAction());
+        // Add more handlers for future actions
```

Comment:
Do you need a handler for the config update action?

Reply:
ConfigUpdate is not exposed via a rest API since its a transport-layer only action. Its intended to be triggered after a successful update to the identity config index to synchronize all of the nodes' in-memory structures of the identity index. 

On a config update each node reloads the configuration by reading it from the index.